### ISSUE TITLE
create version specific STAR/SALMON/KALILSTO index

### ISSUE THREAD

**Issue by hxin** [sidbdri/cookiecutter-de_analysis_skeleton#22]
something like this?    
```
"salmon_version": "0.8.2"
"salmon_index": "salmon_index_{{ cookiecutter.salmon_version }}"
```

**Comment by lweasel** [comment]
In /srv/data/genome/... do as STAR (i.e. versioned indexes).

**Comment by hxin** [comment]
fixed in 94ed1dbdb64a3a7a10a831aec884fce5c753fbb2
